examples/non-resource-url-token-request/non-resource-url/README.md

```diff
@@ -0,0 +1,170 @@
+# non-resource-url example
```

Comment:
hmm, how does this differ from `examples/non-resource-url-token-request/README.md`? Or is this duplication just accidentally?

Reply:
whops I haven't finished this yet it seems